lib/kiev/shoryuken/request_id.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "securerandom"
+
+module Kiev
+  module Shoryuken
+    class RequestId
```

Comment:
Both of this and https://github.com/blacklane/kiev/blob/master/lib/kiev/rack/request_id.rb share some similar code.
I'd suggest to create a general `RequestId` abstract class which encapsulates this code, so that it would be easier to modify the code in future without forgetting to update other middlewares.